mod/ckeditor/views/default/ckeditor/init.php

```diff
@@ -7,12 +7,14 @@
 
 ?>
 <script>
-// This global variable must be set before the editor script loading.
-CKEDITOR_BASEPATH = elgg.config.wwwroot + 'mod/ckeditor/vendors/ckeditor/';
+require(['elgg'], function (elgg) {
+	// This global variable must be set before the editor script loading.
```

Comment:
With this change I'm not sure we can guarantee CKEDITOR_BASEPATH is always defined before ckeditor is loaded..